public_html/i18n/qqq.json

```diff
@@ -0,0 +1,71 @@
+{
+  "@metadata": {
+    "authors": [
+      "Niharika Kohli",
+      "MusikAnimal",
+      "Ryan Kaldari"
+    ]
+  },
+  "name": "Title of the application/tool",
+  "documentation": "Link to documentation in header\n{{Identical|Documentation}}",
+  "feedback": "Link to feedback in header\n{{Identical|Feedback}}",
+  "leaderboard": "Link to leaderboard in header",
```

Comment:
"Leaderboard" may be tricky to understand. You might want to explain this better, like "Link to a list of the top reviewers in header".